ios/FluentUI.Demo/FluentUI.Demo/Demos/AvatarGroupDemoController.swift

```diff
@@ -441,31 +441,35 @@ class AvatarGroupDemoController: DemoTableViewController {
 
     private var avatarCount: Int = 5 {
         didSet {
-            AvatarGroupDemoSection.allCases.filter({ section in
-                return section.isDemoSection
-            }).forEach { section in
-                var avatarGroupsForCurrentSection: [AvatarGroupDemoRow: MSFAvatarGroup] = [:]
-
-                AvatarGroupDemoRow.allCases.filter({ row in
-                    return row.isDemoRow
-                }).forEach { row in
-                    let avatarGroup = demoAvatarGroupsBySection[section]![row]
-                    if oldValue < avatarCount {
-                        let avatarState = avatarGroup?.state.createAvatar()
-                        for index in 0...avatarCount - 1 {
-                            let samplePersona = samplePersonas[index]
-                            avatarState!.image = samplePersona.image
-                            avatarState!.isRingVisible = section.isMixedBorder ? index % 2 == 0 : section.showBorders
-                            avatarState!.primaryText = samplePersona.name
-                            avatarState!.secondaryText = samplePersona.email
+            if avatarCount == 0 && oldValue == 0 {
```

Comment:
Can you use `guard let` for early returns here and in line 482?
**Reference:** 
https://github.com/microsoft/swift-guide/blob/gh-pages/EarlyReturn.md